tests/attacks/lucash-flap-end.mcd

```diff
@@ -2,6 +2,7 @@ STEPS ( DEPLOY-PRELUDE )
 STEPS ( ATTACK-PRELUDE )
 
 transact ADMIN Gem "MKR" . initUser Vow
+transact Vow Vat . hope Flap
```

Comment:
This should really be in the Vow's constructor, is there a reason it can't go there?

Reply:
Ah, actually it is there! I missed that.